node/test/lib/function-actions.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
```

Comment:
Do we need that refactor?

Reply:
No I don't think we need it anymore, I can back this out 👍